axelrod/strategies/shortmem.py

```diff
@@ -0,0 +1,43 @@
+from axelrod import Actions, Player
+from axelrod.actions import Action
+
+C, D = Actions.C, Actions.D
+
+class ShortMem(Player):
+    """
+    A player starts by always cooperating for the first 10 moves.
+    
+    The opponent answers are stored in the memory. The memory is FIFO and the maximum size of the memory is 10 results. From the tenth round on, the program analyzes the memory, and compare the number of defects and cooperates of the opponent, based in percentage. If cooperation occurs 30% more than defection, it will cooperate.
+    If defection occurs 30% more than cooperation, the program will defect. Otherwise, the program follows the TFT algorithm.
+    
+    """
+
+    name = 'ShortMem'
+    classifier = {
+        'memory_depth': 10,
+        'stochastic': False,
+        'makes_use_of': set(),
+        'long_run_time': False,
+        'inspects_source': False,
+        'manipulates_source': False,
+        'manipulates_state': False
+    }
+
+    @staticmethod
+    def strategy(opponent: Player) -> Action:
+        
+        memoryDepth = self.classifier['memoryDepth']
+        array = self.history[:-11:-1]
```

Comment:
As order doesn't matter you can just take: `self.history[-10:]`.

```
>>> a = list(range(20))
>>> a[-10:]
[10,11,12,13,14,15,16,17,18,19]
```

However note that I think you mean for this to be `opponent.history`.